aced_submission/grip_load.py

```diff
@@ -237,23 +237,27 @@ def delete_project(grip_service: str, graph_name: str, project_id: str, output:
     return json_data
 
 
-def get_project_data(grip_service: str, graph_name: str, project_id: str, output: dict, access_token: str) -> Generator[dict, None, None]:
+def get_project_data(grip_service: str, graph_name: str, project_id: str, output: dict, access_token: str, chunk_size: int) -> Generator[dict, None, None]:
     """Retrieves all of the data for a given project id on a given graph"""
     response = requests.get(
         f"http://{grip_service}:8201/{NGINX_PATH}/{graph_name}/get-vertices/{project_id}",
         headers={"Authorization": f"bearer {access_token}"}
     )
 
-    def stream_res(response):
-        for result in response.iter_lines(chunk_size=None):
-            try:
-                result_dict = orjson.loads(result.decode())
-            except Exception as e:
-                print("Failed to decode: %s", result)
-                raise e
-            yield result_dict["data"]
-
-    return stream_res(response)
+    for result in response.iter_lines(chunk_size=chunk_size):
+        try:
+            result_dict = orjson.loads(result.decode())
+            # Elastic doesn't like '_' prefix fields. Need to sanitize them before passing to elastic
+            result_dict["id"] = result_dict["_id"]
```

Comment:
writing to "label" makes sense cause there's no label, what's the difference between "_id" and "id"? Why do we have to overwrite "id"?

Reply:
How do we know that the value of "_id" will always be the same as "id"? And if GRIP is always writing it this way, what's the point of writing the same id and instead just running `del result_dict["_id"]`?

Maybe I'm just confused between the usage of `_id` and `id`, my understanding is that the `_id` is GRIP and `id` is FHIR.

Reply:
yeah this line might be redundant ```result_dict["id"] = result_dict["_id"]``` and pop might be all that is needed here.


I'm hesitant to change this, get an error and then have to change it back. Make of a note of this for next time something breaking needs to be changed in submission. This is a small detail, not worth cutting a new release for